auth/strategies.js

```diff
@@ -0,0 +1,55 @@
+const passport = require('passport');
+const {BasicStrategy} = require('passport-http');
+const {Strategy: JwtStrategy, ExtractJwt} = require('passport-jwt');
```

Comment:
might want to add code comment about renaming with destructuring since this will not be familiar to a lot of folks at this point.